RELEASE_PROCESS_OPENSEARCH.md

```diff
@@ -0,0 +1,322 @@
+- [OpenSearch Releas Process](#opensearch-release-process)
+  - [Release Terminology and Knowledge Center](#release-terminology-and-knowledge-center)
+    - [Types of Manifests](#types-of-manifests)
+    - [Build Workflows](#build-workflows)
+    - [Release Workflows](#release-workflows)
+    - [Creating a New Version](#creating-a-new-version)
+  - [Release Manager](#release-manager)
+  - [Release issue](#release-issue)
+    - [Release Issue Update](#release-issue-update)
+    - [Component Issues and Campaigns](#component-issues-and-campaigns)
+  - [Version Increment](#version-increment)
+     - [Version Increment Core](#version-increment-core)
+     - [Release Branch](#release-branch)
+     - [Plugins Version Increment](#plugins-version-increment)
+ - [Distribution Build](#distribution-build)
+ - [Release Candidate](#release-candidate)
+ - [Final Release](#final-release)
+    - [Pre-Release](#pre-release)
+    - [Release Notes](#release-notes)
+    - [Release](#release)
+    - [Post-Release](#post-release)
+
+## OpenSearch Release Process
+
+### Release Terminology and Knowledge Center
+
+#### Types of Manifests
+
+
+- Input Manifest: Uses as an initial input to drive the build workflow. These manifests are [located](https://github.com/opensearch-project/opensearch-build/tree/main/manifests) in build repo and gets created by [manifest workflow](https://github.com/opensearch-project/opensearch-build/tree/main/src/manifests_workflow).
+Once the manifest is created the maintainers of the build repo approves and merges it to `main` branch. The entire build repo runs on single branch `main`.
```

Comment:
This can be discussed further but with more automation (a release wizard)  a RM does even has to care about this anymore, this explain how the input manifest works and added to `main` branch. 

A proposal (but might take a long way) is a RM can just cut a tag or start with a draft release in GH which kicks in a workflow that builds an RC and sets the stage for release. Once the PMC committee click approve, the release workflow kicks in that does the release and post release activities. So all the RM here has to be do is validate the RC.